systest/Makefile

```diff
@@ -355,9 +336,8 @@ tempest_12.1.1_undercloud_vlan:
 	export EXCLUDE_FILE=$@.yaml ;\
 	$(MAKE) -C . run_tests
 
-run_tests: install_tlc install_test_infra
-	$(MAKE) -C . setup_tlc_session
+run_tests:
+	$(MAKE) -C . microservice_setup_tlc_session
 	$(MAKE) -C . configure_test_infra
 	$(MAKE) -C . run_neutron_lbaas
-	$(MAKE) -C . run_agent_transfers
```

Comment:
We can remove them or exclude them. Either way is fine, since we don't yet know the expectation of these tests.